admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -1335,9 +1335,12 @@ window.imagesProduct = (function () {
       dropZoneElem.css('height', '');
       const closedHeight = dropZoneElem.outerHeight();
       const realHeight = dropZoneElem[0].scrollHeight;
-      dropZoneElem.css('height', oldHeight);
 
-      return (realHeight > closedHeight);
+      if (oldHeight !== '0px') {
+        dropZoneElem.css('height', oldHeight);
+      }
+
+      return (realHeight > closedHeight) && dropZoneElem.find('.dz-preview:not(.openfilemanager)').length;
```

Comment:
If `dropZoneElem.find('.dz-preview:not(.openfilemanager)')` cannot be found, is it OK to call `.length()` on it?

Reply:
Yep because JQuery returns an array of elements, the array length will be 0 with a previousElement key

Reply:
Could you move the selector to a selector map ?

Reply:
@Progi1984 nop, it's from legacy folder, I won't create a map for this, it's out of the scope of the PR and inside a folder that should be deprecated in the future